src/Model/GameManager.php

```diff
@@ -82,4 +80,24 @@ public function selectOneGameById(int $id): array
 
         return $statement->fetch();
     }
+
+    public function selectMyGames(int $id): array
+    {
+        $query = 'SELECT * FROM game as g WHERE g.id_owner=:id';
+        $statement = $this->pdo->prepare($query);
+        $statement->bindValue(':id', $id, \PDO::PARAM_INT);
+        $statement->execute();
+
+        return $statement->fetchAll();
+    }
+
+    public function returnGame(int $id)
```

Comment:
Rename this function "giveBackGame" or something like that but please avoid return in the name. Also add a return type please

Reply:
Correction done.